lib/service_sign_in/README.md

```diff
@@ -0,0 +1,57 @@
+# How to publish Service Sign-in pages
+
+If a Content Designer adds a new page via Github, there's a good chance they'll be an external contributor.
+
+If this is not the case feel free to skip 'How to merge a change from an external contributor' steps.
+
+The examples below are from a [release from the Start Pages team](https://github.com/alphagov/publisher/pull/687)
+
+## How to merge a change from an external contributor
+
+As described in the [Pull Request merging RFC](https://github.com/alphagov/govuk-rfcs/blob/2135a4c9363a499a7182e7594d6c4054422e76a1/rfc-052-pull-request-merging-process.md#a-change-from-an-external-contributor) we have a slightly different process for contributions that are not from people in the alphagov organisation.
+
+The main things to note are that we require extra effort in reviews to make sure we don't accidentally merge something malicious.
+
+1. Get a second reviewer, this can be bi-passed for these pages by going to talk to the Content Designer and confirming their Github account is the one raising the request.
+2. Get Continuous Integration to run their Pull Request so that the build will pass.
+
+### Merging external contribution: Getting the build to pass
+
+You'll need to pull their branch and push it yourself to trigger the build, this relies on the fact that their commit will be the same in the branch you push up. You can then remove this branch after this process is finished.
+
+```bash
```

Comment:
Can this section be moved to the developer docs? I wanted to do this a couple of weeks but I forgot how. Would help a lot.

Reply:
Sounds good 👍